src/Numerics/DGMethods/DGModel_kernels.jl

```diff
@@ -90,9 +89,9 @@ fluxes, respectively.
         ngradlapstate = number_states(balance_law, GradientLaplacian())
         nhyperviscstate = number_states(balance_law, Hyperdiffusive())
 
-        Nq = N + 1
-
-        Nqk = dim == 2 ? 1 : Nq
+        Nq = info.Nq[1]
+        Nq2 = info.Nq[2]
+        Nqk = info.Nqk
```

Comment:
For the sake of consistence, should we now rename this `Nq` as `Nq1`?

Reply:
Great idea --- once we get everything passing tests, we should do another sweep and clean up (including better variable names where appropriate!)

Reply:
Did you want to address this comment in this PR?

Reply:
I volunteered to do it!  I just didn't have the chance last night. I am going to do it now

Reply:
👍